.github/workflows/SpeakerView-builds.yml

```diff
@@ -35,6 +35,32 @@ jobs:
           name: windows
           path: build/windows
 
+  export-linux-arm:
+    name: Linux ARM Export
+    runs-on: ubuntu-24.04
```

Comment:
It's ok if it's not ubuntu-24.04-arm ?

Reply:
just tested and the build starts! it has visual glitches but I'd say they are outside the scope of this MR as they are likely due to the vulkan rendering engine in godot